project/Dependencies.scala

```diff
@@ -23,42 +22,55 @@ object Dependencies {
   )
 
   object V {
-    val yaml       = "1.10"
-    val http       = "4.3.3"
+    val yaml = "1.10"
+    val http = "4.3.3"
     object specs2 {
-      val _29   = "1.12.1"
-      val _210  = "1.14"
-      val _211  = "2.3.13"
+      val _29  = "1.12.1"
+      val _210 = "1.14"
+      val _211 = "2.3.13"
     }
-    val scalaCheck = "1.10.0"
-    val scalaUtil  = "0.1.0"
-    val junit      = "4.11"
-    val json       = "20140107"
-    val json4s     = "3.2.9"
+    val scalaCheck   = "1.10.0"
+    val scalaUtil    = "0.1.0"
+    val junit        = "4.11"
+    val json         = "20140107"
+    val json4s       = "3.2.9"
+    val logback      = "1.2.3"
```

Comment:
Are the additional dependencies required for 0.3.1? If yes, there should be issues for them and they should be reflected in the CHANGELOG.